src/Microsoft.VisualBasic.Core/src/Microsoft/VisualBasic/Interaction.vb

```diff
@@ -26,6 +126,23 @@ Namespace Microsoft.VisualBasic
 #End If
         End Sub
 
+        Public Function InputBox(ByVal Prompt As String, Optional ByVal Title As String = "", Optional ByVal DefaultResponse As String = "", Optional ByVal XPos As Integer = -1, Optional ByVal YPos As Integer = -1) As String
+            Return DirectCast(InvokeMethod("InputBox", Prompt, Title, DefaultResponse, XPos, YPos), String)
+        End Function
+
+        Public Function MsgBox(ByVal Prompt As Object, Optional ByVal Buttons As MsgBoxStyle = MsgBoxStyle.OkOnly, Optional ByVal Title As Object = Nothing) As MsgBoxResult
+            Return DirectCast(InvokeMethod("MsgBox", Prompt, Buttons, Title), MsgBoxResult)
+        End Function
+
+        Private Function InvokeMethod(methodName As String, ParamArray args As Object()) As Object
+            Dim type As Type = type.GetType("Microsoft.VisualBasic._Interaction, Microsoft.VisualBasic, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a", throwOnError:=True)
```

Comment:
We'll need a new helper type in Microsoft.VisualBasic. I'm assuming the type is public but with a special name, but perhaps the type should be internal.

Reply:
I assume this new type is being added concurrently with this change and will be in 3.0?

Reply:
The `_Interaction` type will be added to Microsoft.VisualBasic.dll in dotnet/winforms. Both changes are post-3.0. I'll hold off merging this PR until that type is available.

Reply:
Are you OK with a `TypeLoadException` when someone calls this from a console app that doesn't have that type?  You might want to remap that to something else (like `PlatformNotSupportedException`).

Reply:
Seems that avoiding the `TypeLoadException` is desirable.